src/resources/rmd/hooks.R

```diff
@@ -124,6 +124,11 @@ knitr_hooks <- function(format, resourceDir, handledLanguages) {
     options
   }
 
+  opts_hooks[["renderings"]] <- function(options){
+    options$fig.show = 'hold'
+    options
+  }
```

Comment:
I would add a comment in there so that later us or future quarto-dev remembers that this option and hook is about dark/light feature. 

But the commits and github issue also explain it, so git Blame will have the info. 

So as you want !